components/export/api/src/main/java/org/phenotips/export/internal/DataToCellConverter.java

```diff
@@ -66,34 +68,38 @@
 
     public static final Integer charactersPerLine = 100;
 
+    private String translate(String term_id)
```

Comment:
Instead of getting the component each time and having a new method just for this, you should make the `translationManager` a class field, initialized in a constructor.